packages/terra-functional-testing/src/reporters/wdio-accessibility-reporter.js

```diff
@@ -0,0 +1,123 @@
+const chalk = require('chalk');
+const WDIOReporter = require('@wdio/reporter').default;
+
+class AccessibilityReporter extends WDIOReporter {
+  /**
+   * Indents a string.
+   * @param {string} string - The string to indent.
+   * @param {number} indent - The number of spaces to indent the string.
+   */
+  static indent(string, indent) {
+    if (indent > 0) {
+      return string.replace(/^(?!\s*$)/gm, Array(indent + 1).join(' '));
+    }
+
+    return string;
+  }
+
+  constructor(options) {
+    super(options);
```

Comment:
Shall we defined the outputDir and logFile option to pass to super ? It seems there is only one option for output files? maybe, which option 2 from this comment: https://github.com/cerner/terra-toolkit/pull/456/files#r513483163

Reply:
From the discussion in our WDIO 6 sync I'd say this PR is not going to write out to a file, but it is something we can discuss and circle back on down the road if we think it would be beneficial